samples/matter/light_bulb/src/app_task.cpp

```diff
@@ -103,10 +117,18 @@ int AppTask::Init()
 	ConfigurationMgr().LogDeviceConfig();
 	PrintOnboardingCodes(chip::RendezvousInformationFlags(chip::RendezvousInformationFlag::kBLE));
 
-#if defined(CONFIG_CHIP_NFC_COMMISSIONING)
+#ifdef CONFIG_CHIP_NFC_COMMISSIONING
```

Comment:
Actually I think that line should be removed, as ChipEventHandler should be available to use even with NFC disabled

Reply:
Right